var/spack/repos/builtin/packages/bueno/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Bueno(PythonPackage):
+    """Bueno: Well-Provenanced Benchmarking"""
+
+    homepage    = "https://lanl.github.io/bueno"
+    url         = "https://github.com/lanl/bueno/archive/refs/tags/v0.0.1.tar.gz"
```

Comment:
Basics LGTM.

Do you intend to add a version directive for `v0.0.1`?

Reply:
FWIW. If so, you'll want it to be:

```
    version('0.0.1', sha256='28775bcba13dc8bb89f19ee0d7a8f86137efcf29174d31a65e7611dd85a4ddf9')
```

Reply:
Yes, once there is a tagged release that supports being installed with Spack. Right now only `master` works, since the `v0.0.1` release was bundling dependencies as tarballs and had hardcoded paths in its `setup.py`. PR https://github.com/lanl/bueno/pull/22 fixed this.